PluralKit.Core/Database/Views/DatabaseViewsExt.cs

```diff
@@ -44,7 +44,29 @@ public static Task<IEnumerable<ListedMember>> QueryMemberList(this IPKConnection
             
             return conn.QueryAsync<ListedMember>(query.ToString(), new {system, filter = opts.Search, groupFilter = opts.GroupFilter});
         }
-        
+
+        //r1 and r2 are needed in order to return the "seen" value before it's updated
+        public static IAsyncEnumerable<PKReminder> QueryUnseenReminders(this IPKConnection conn, MemberId member) => conn.QueryStreamAsync<PKReminder>(@"
```

Comment:
This code is pretty much duplicated. Can you add a `bool seen = false` parameter on QueryReminders, or something to that effect?